scripts/issueEmailGenerator.js

```diff
@@ -17,207 +17,226 @@ const optionDefinitions = [
+    currentIssue.devOfTheWeek
+      ? `# Dev of the Week 
+      
+  <img alt="${currentIssue.devOfTheWeek.name}" src="${getAssetURL(
+          currentIssue.id,
+          currentIssue.devOfTheWeek.profileImg
+        )}" style="width:200px;"/> 
     
-${tool.description}
-
-*by ${tool.authors.map(author => author.Name).join(', ')}*
-`
-  )
-  .join('\n')}
-
-___
-
-# Tech Talks
-
-[**${currentIssue.talks[0].title}**](${currentIssue.talks[0].url})
-
-${currentIssue.talks[0].url}
-
-${currentIssue.talks[0].description}
-
-___
-
-# Quiz
-
-### ${currentIssue.quiz.question}
-
-\`\`\`${currentIssue.quiz.CodeSnippet.language}
-${currentIssue.quiz.CodeSnippet.code}
-\`\`\`
-
-${currentIssue.quiz.Option.map(
-  option => `<a href="https://scriptified.dev/issues/${currentIssue.id}?section=quiz&option=${option.option_id}" style="text-decoration:none;">
-<div style="margin: 12px 0px; border: 1px solid gray; padding: 16px; background: #F2F3F5;">
-	${option.text}
-</div>
-</a>
-`
-).join('\n')}
-
-
-___
-
-# This week in GIF
-
-![${currentIssue.gif.caption}](${getAssetURL(currentIssue.id, currentIssue.gif.gifURL)})
-
-<center>${currentIssue.gif.caption}</center>
-
----
-
-Liked this issue? [Share on Twitter](https://twitter.com/intent/tweet?text=${encodeURIComponent(`Have a look at issue #${currentIssue.id} of Scriptified.
-
-Subscribe to @scriptified_dev for more.`)}&url=${encodeURIComponent(
-    `https://scriptified.dev/issues/${currentIssue.id}`
-  )}) or [read previous issues](https://scriptified.dev/issues).
-`;
+  ## ${currentIssue.devOfTheWeek.name} 
+  ${currentIssue.devOfTheWeek.bio} 
+    
+  ${Object.keys(currentIssue.devOfTheWeek)
+    .filter(key => PROFILE_KEYS.includes(key) && currentIssue.devOfTheWeek[key] !== null)
+    .map(profile => `[${PROFILE_TYPES[profile]}](${currentIssue.devOfTheWeek[profile]})`)
+    .join(' | ')}
+  
+  ___`
+      : ''
+  }
+  
+  # Tools
+  
+  ${currentIssue.tools
+    .map(
+      tool =>
+        `[**${tool.name}**](${tool.url})
+      
+  ${tool.description}
+  
+  ${getAuthors(tool?.authors)}
+  `
+    )
+    .join('\n')}
+  
+  ___
+  
+  # Tech Talks
+  
+  [**${currentIssue.talks[0].title}**](${currentIssue.talks[0].url})
+  
+  ${currentIssue.talks[0].url}
+  
+  ${currentIssue.talks[0].description}
+  
+  ${getAuthors(currentIssue.talks?.[0]?.authors)}
+  
+  ___
+  
+  # Quiz
+  
+  ### ${currentIssue.quiz.question}
+  
+  \`\`\`${currentIssue.quiz.CodeSnippet.language}
+  ${currentIssue.quiz.CodeSnippet.code}
+  \`\`\`
+  
+  ${currentIssue.quiz.Option.map(
+    option => `<a href="https://scriptified.dev/issues/${currentIssue.id}?section=quiz&option=${option.option_id}" style="text-decoration:none;">
+  <div style="margin: 12px 0px; border: 1px solid gray; padding: 16px; background: #F2F3F5;">
+    ${option.text}
+  </div>
+  </a>
+  `
+  ).join('\n')}
+  
+  
+  ___
+  
+  # This week in GIF
+  
+  [${currentIssue.gif.caption}](${process.env.SITE_URL}issues/${options.issueNumber}?section=gif)
```

Comment:
@gupta-ji6 Why did we remove the gif image and replaced with Link?

Reply:
in #69, we replaced all the gifs media with videos & videos weren't supported to be sent in Buttondown if I remember correctly.

Reply:
Oh yeah, can't do video in emails. I guess we would need to still figure out a way to keep two versions, video for the web and gif for the email, because just adding a link doesn't look good. Although this can be done in a separate PR, so let's bring this in for now